network/horizon/admin-guide/error-handling.mdx

```diff
@@ -0,0 +1,182 @@
+| `FAILED`                | -1   | One of the operations failed (see [List of Operations](/docs/learn/fundamentals/list-of-operations) for errors)                           |
+| `TOO_EARLY`             | -2   | Ledger `closeTime` before `minTime` value in the transaction                                                                              |
+| `TOO_LATE`              | -3   | Ledger `closeTime` after `maxTime` value in the transaction                                                                               |
+| `MISSING_OPERATION`     | -4   | No operation was specified                                                                                                                |
+| `BAD_SEQ`               | -5   | Sequence number does not match source account                                                                                             |
+| `BAD_AUTH`              | -6   | Too few valid signatures / wrong network                                                                                                  |
+| `INSUFFICIENT_BALANCE`  | -7   | Fee would bring account below minimum balance; see our section on [Lumens](/docs/learn/fundamentals/lumens#minimum-balance) for more info |
+| `NO_ACCOUNT`            | -8   | Source account not found                                                                                                                  |
+| `INSUFFICIENT_FEE`      | -9   | Fee is too small; see our section on [Fees](/docs/learn/fundamentals/fees-resource-limits-metering) for more info                         |
+| `BAD_AUTH_EXTRA`        | -10  | Unused signatures attached to transaction                                                                                                 |
+| `INTERNAL_ERROR`        | -11  | An unknown error occurred                                                                                                                 |
+| `NOT_SUPPORTED`         | -12  | The transaction type is not supported                                                                                                     |
+| `FEE_BUMP_INNER_FAILED` | -13  | The fee bump inner transaction failed                                                                                                     |
+| `BAD_SPONSORSHIP`       | -14  | The sponsorship is not confirmed                                                                                                          |
+
+### Insufficient fees and surge pricing
+
+See the [Fees section](/docs/learn/fundamentals/fees-resource-limits-metering)
+
+### Rate limiting
+
+If you’re using SDF’s public Horizon instance, you may get a `429 Too Many Requests` error when exceeding the rate limits. If you’re encountering this frequently, it may be time to deploy your own Horizon instance!
```

Comment:
We removed all references to SDF horizon URL in docs, I think I'd prefer to leave out the mention of it explicitly here as well.

```suggestion
Horizon may rate limit requests and return `429 Too Many Requests` error when exceeding the rate limits. If using your own instance, you may want to either increase or disable rate limiting. If you're using a third-party Horizon instance, you may want to deploy your own to have more control over this configuration.
```